cohort-util/src/test/java/com/ibm/cohort/valueset/ValueSetUtilTest.java

```diff
@@ -100,10 +105,23 @@ public void testArtifactCreation() throws IOException {
 		String valueSetInput = "src/test/resources/2.16.840.1.113762.1.4.1114.7.xlsx";
 		File tempFile = new File(valueSetInput);
 		byte[] byteArrayInput = Files.readAllBytes(Paths.get(tempFile.getAbsolutePath()));
-		ValueSetArtifact artifact = ValueSetUtil.createArtifact(new ByteArrayInputStream(byteArrayInput));
+		ValueSetArtifact artifact = ValueSetUtil.createArtifact(new ByteArrayInputStream(byteArrayInput), null);
 		assertEquals("testValueSet", artifact.getFhirResource().getId());
 		assertEquals("http://cts.nlm.nih.gov/fhir/ValueSet/testValueSet", artifact.getUrl());
 		assertEquals("Value Set For Testing Uploads", artifact.getName());
 	}
 
+	@Test
+	public void testMapCreation() throws IOException {
+		String codeSystemInput = "src/test/resources/codeSystemOverride.txt";
+		Map<String, String> expectedMap = new HashMap<>();
+		expectedMap.put("tomato", "http://abc.com");
+		expectedMap.put("excedrin", "http://nih.no.com");
+		Map<String, String> codeSystemMap = ValueSetUtil.getMapFromInputStream(new FileInputStream(new File(codeSystemInput)));
+		assertEquals(expectedMap.size(), codeSystemMap.size());
+		assertEquals(expectedMap.entrySet(), codeSystemMap.entrySet());
+		assertTrue(expectedMap.values().containsAll(codeSystemMap.values()));
+		assertTrue(codeSystemMap.values().containsAll(expectedMap.values()));
```

Comment:
Pedantic: I could be wrong, but I think the `containsAll` checks are a tad overkill.  I don't think these will ever fail while the `entrySet()` equality check passes.  You also may be able to get away with just one `assertEquals()` that compares the maps.

Reply:
I didn't realize I'd left all three in here, definitely leftover from playing with a different concept of testing at one point.